nb-extension/python/declarativewidgets/extensionapp.py

```diff
@@ -177,35 +187,90 @@ def start(self):
 
         self.log.info("Deactivating jupyter_declarativewidgets JS notebook extensions")
         self.section = "notebook"
-        self.disable_nbextension("urth_widgets/js/main")
+        self.disable_nbextension("declarativewidgets/js/main")
 
         self.log.info("Done. You may need to restart the Jupyter notebook server for changes to take effect.")
 
 
+class ExtensionQuickSetupApp(BaseNBExtensionApp):
+    """Installs and enables all parts of this extension"""
+    name = "jupyter declarativewidgets quick-setup"
+    version = __version__
+    description = "Installs and enables all features of the jupyter_declarativewidgets extension"
+
+    def start(self):
+        self.argv.extend(['--py', 'declarativewidgets'])
+
+        from notebook import serverextensions
+        install = serverextensions.EnableServerExtensionApp()
+        install.initialize(self.argv)
+        install.start()
+        from notebook import nbextensions
+        install = nbextensions.InstallNBExtensionApp()
+        install.initialize(self.argv)
+        install.start()
+        enable = nbextensions.EnableNBExtensionApp()
+        enable.initialize(self.argv)
+        enable.start()
+
+class ExtensionQuickRemovalApp(BaseNBExtensionApp):
+    """Disables and uninstalls all parts of this extension"""
+    name = "jupyter declarativewidgets quick-remove"
+    version = __version__
+    description = "Disables and removes all features of the jupyter_declarativewidgets extension"
+
+    def start(self):
+        self.argv.extend(['--py', 'declarativewidgets'])
+
+        from notebook import nbextensions
+        enable = nbextensions.DisableNBExtensionApp()
+        enable.initialize(self.argv)
+        enable.start()
+        install = nbextensions.UninstallNBExtensionApp()
+        install.initialize(self.argv)
+        install.start()
+        from notebook import serverextensions
+        install = serverextensions.DisableServerExtensionApp()
+        install.initialize(self.argv)
+        install.start()
+
 class ExtensionApp(Application):
     '''CLI for extension management.'''
     name = u'jupyter_declarativewidgets extension'
     description = u'Utilities for managing the jupyter_declarativewidgets extension'
     examples = ""
 
-    subcommands = dict(
-        install=(
-            ExtensionInstallApp,
-            "Install the extension."
-        ),
-        activate=(
-            ExtensionActivateApp,
-            "Activate the extension."
-        ),
-        deactivate=(
-            ExtensionDeactivateApp,
-            "Deactivate the extension."
-        ),
-        installr=(
-            ExtensionInstallRApp,
-            "Install the R extension"
-        )
-    )
+    subcommands = dict(installr=(
+        ExtensionInstallRApp,
+        "Install the R extension"
+    ))
+
+    if _new_extensions:
```

Comment:
Bah. More old stuff lurking in my dev tree when I built the package. Trying again.